### DISCUSSION TITLE
Tailwind algorithm for palette generation

### DISCUSSION THREAD

**Discussion by Ericlm** [tailwindlabs/tailwindcss#10570] [Help] [answered]
Hello there ! I would like to know where Tailwinds' colors palettes comes from ; is it an algorithm, or are they manually chosen for each color ?

Thank you!

**Comment by adamwathan** [comment, marked as answer]
Hey! Manually chosen, just weeks spent in Figma designing example interfaces and testing and tweaking things by eye until it felt right.

**Reply by Ericlm** [reply]
Alright, thank you for your answer 😊
